srcpkgs/cagebreak/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'cagebreak'
+pkgname=cagebreak
+version=1.9.1
+revision=1
+wrksrc="${pkgname}"
+build_style=meson
+configure_args="--buildtype=release -Dman-pages=true $(vopt_bool xwayland xwayland)"
+conf_files="/etc/xdg/cagebreak/config"
+hostmakedepends="pkg-config wayland-devel scdoc"
+makedepends="wlroots-devel cairo-devel pango-devel libevdev-devel"
+depends="$(vopt_if xwayland xorg-server-xwayland)"
```

Comment:
What happens when cagebreak is built with x Wayland support, but doesn't find xwayland on the system? Is there a config option that can disable xwayland even if it was enabled in the build?

Reply:
Sorry for the late response. I checked the manual and there is not an option to disable xwayland if it could not be found. What would you suggest is the best approach, disable xwayland altogether?